### ISSUE TITLE
Error could not convert string to float: 'SP13'

### ISSUE THREAD

**Issue by 0xMar** [xtero/CrunchyREroll#14]
Hi upon attempting to access **watchlist** folder, I encountered the following error: 

`ValueError: could not convert string to float: 'SP13'
`

This issue arises as an instance in the anime series One Piece.

Kodi log:
```
2024-04-14 10:47:38.802 T:51090   error <general>: [Crunchyroll.support] could not convert string to float: 'SP13'
                                                   Traceback (most recent call last):
                                                     File "/home/victor/.kodi/addons/script.module.codequick/lib/codequick/support.py", line 306, in run_callback
                                                       redirect = parent_ins(route, arg_params, self.callback_params)
                                                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                                                     File "/home/victor/.kodi/addons/script.module.codequick/lib/codequick/route.py", line 195, in __call__
                                                       results = super(Route, self).__call__(route, args, kwargs)
                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                                                     File "/home/victor/.kodi/addons/script.module.codequick/lib/codequick/script.py", line 169, in __call__
                                                       return route.function(self, *args, **kwargs)
                                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                                                     File "/home/victor/.kodi/addons/plugin.video.crunchyreroll/resources/lib/main.py", line 64, in watchlist
                                                       episodes, next_link = cr.get_watchlist(start)
                                                                             ^^^^^^^^^^^^^^^^^^^^^^^
                                                     File "/home/victor/.kodi/addons/plugin.video.crunchyreroll/resources/lib/client.py", line 99, in get_watchlist
                                                       res.append(Episode(item, playhead))
                                                                  ^^^^^^^^^^^^^^^^^^^^^^^
                                                     File "/home/victor/.kodi/addons/plugin.video.crunchyreroll/resources/lib/model.py", line 31, in __init__
                                                       self.number = int(float(self.number_str))
                                                                         ^^^^^^^^^^^^^^^^^^^^^^
                                                   ValueError: could not convert string to float: 'SP13'
```

- Version v1.0.3

**Comment by xtero** [comment]
Crap, my previous fix was really dirty. I'll find something better when I come back from my holidays.

**Comment by xtero** [comment]
fixed in version v1.0.4
